_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
```

Comment:
@maelle Thank you for contributing this post! The last of the unconf packages to be featured before we post "the Roads Not Taken". 

I'll include my comments to consider.